docs/tutorials/developer-journey/level-3/3.3-certified-data.mdx

```diff
@@ -199,12 +199,12 @@ git clone https://github.com/krpeacock/certified-cache
 cd certified-cache
 ```
 
-Then, install the following packages with Mops:
+Upgrade your Node.js version to [v22.12.0](https://nodejs.org/en/download). Then, install the following packages with Mops:
```

Comment:
Do you know why this specific version? It's a few versions behind the latest Node.js version 23.6.0.